packages/artifact_proxy/lib/config.dart

```diff
@@ -8,62 +8,50 @@ const config = ProxyConfig(
   },
 );
 
+// On the assumption artifact layouts don't change very often for Flutter.
+class _EngineMapping37 extends EngineMapping {
```

Comment:
Is the 37 supposed to represent Flutter 3.7.x?

Reply:
Yeah. I tried 3_7 and the linter got mad at me.

Reply:
Maybe `_EngineMapping3_7` instead?

Reply:
Will change and update the linter. :)

Reply:
Sniped 🙂 

I'd vote to override the linter on this one.

Reply:
Overriden. 🤣  PTAL.